lens-macro-impl/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "lens-macro-impl"
```

Comment:
I wonder if we can't put this crate _inside_ the main `lens-macro` crate's root? That would clean things up somewhat.